### ISSUE TITLE
实验里一处不合理的地方

### ISSUE THREAD

**Issue by kaqiz** [kunzhan/M-ILBO#1]
实验的代码里
` if val_acc >= best_val_acc: 
                best_val_acc = val_acc
                if test_acc > eval_acc:
                    eval_acc = test_acc`
这里是不是不太合理，正常应该是取验证集最好的一次保存，然后算出测试集的结果，这样加上 if test_acc > eval_acc的话不就相等于面相答案调参了吗

**Comment by kunzhan** [comment]
validation set can be used to tune the hyper-parameters.

**Comment by kaqiz** [comment]
> validation set can be used to tune the hyper-parameters.

![image](https://github.com/kunzhan/M-ILBO/assets/30365983/29da0085-3c4d-4790-b0a5-948bdee6a3b1)
I have got the reply of the author of CCA-SSG, The method is not good

**Comment by kunzhan** [comment]
https://github.com/hengruizhang98/CCA-SSG/issues/2

I read it.
most of our results are from CCA-SSG too.
